draft-js-counter-plugin/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "draft-js-counter-plugin",
+  "version": "0.0.5",
```

Comment:
Can you start from version 0.0.1? I'm not entirely sure, but I think keeping all the components & versions aligned will